### PR TITLE
Create quickstart.py not related to docker

### PR DESCRIPTION
**Pull request by AbdulABHILASHA**
I felt like we really needed an quickstart file that is not related to docker.
# **I DID IT ON MY OWN AND HAD NO HELP DOING IT THANKS BYEBYE**

If you feel like it is not useful please reconsider that someone else myght have the sabe problem as myself had once couldnt not use the bott क्योंकि it does not contain a regular quickstart i dont even know what docker means

### CODE REVIEW

**Comment by AbdulABHILASHA** [comment]
fuhrermore, it says in the READM that it contains a quickstarter and id does nott.............


#  This is also implicit in all the exmples and quickstart.py

**Comment by timgrossmann** [comment]
@AbdulABHILASHA Thank you for mentioning that. <strike>I'll add a more simple quickstart file to the project.
I think yours is too specific already.</strike>

**Comment by AbdulABHILASHA** [comment]
Want me to fix it how so?

File: quickstart.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from instapy import InstaPy
+
+insta_username = ''
+insta_password = ''
+
+InstaPy(username=insta_username, password=insta_password, use_firefox=False, page_delay=25)\
```
Comment (timgrossmann) [inline quickstart.py]: remove use_firefox and the page_delay

File: quickstart.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  .login()\
+  .set_switch_language(False)\
```
Comment (timgrossmann) [inline quickstart.py]: remove this line

**Comment by AbdulABHILASHA** [comment]
i want to fex my mistakes

**Review by timgrossmann — CHANGES_REQUESTED** [review]

**Review by timgrossmann — APPROVED** [review]

**Comment by AbdulABHILASHA** [comment]
Done

**Comment by AbdulABHILASHA** [comment]
thankx i feel helped other

**Pull request merged**